actors/builtin/miner/miner_state.go

```diff
@@ -725,22 +724,18 @@ func (st *State) SaveVestingFunds(store adt.Store, funds *VestingFunds) error {
 //
 
 func (st *State) AddPreCommitDeposit(amount abi.TokenAmount) {
-	newTotal := big.Add(st.PreCommitDeposits, amount)
-	AssertMsg(newTotal.GreaterThanEqual(big.Zero()), "negative pre-commit deposit %s after adding %s to prior %s",
```

Comment:
Removing these makes me nervous. I guess it's ok as long as we are sure that every call modifying the balance allocations runs CheckBalanceInvariants.  I would prefer returning an error here as we do in `AddLockedFunds` so that we don't have to worry about missed checks allowing error states to survive across message calls.

Reply:
Done